scipy/stats/tests/test_binned_statistic.py

```diff
@@ -437,15 +437,16 @@ def test_dd_multi_values(self):
         v = self.v
         w = self.w
 
-        stat1v, edges1v, bc1v = binned_statistic_dd(X, v, np.std, bins=8)
-        stat1w, edges1w, bc1w = binned_statistic_dd(X, w, np.std, bins=8)
-        stat2, edges2, bc2 = binned_statistic_dd(X, [v, w], np.std, bins=8)
-
-        assert_allclose(stat2[0], stat1v)
-        assert_allclose(stat2[1], stat1w)
-        assert_allclose(edges1v, edges2)
-        assert_allclose(edges1w, edges2)
-        assert_allclose(bc1v, bc2)
+        for stat in ["count", "sum", "mean", "std", "min", "max", "median",
+                     np.std]:
```

Comment:
why the `np.std` is here,  using as an example of passing in `callable`, or is there any other reason?

Reply:
Because, I think we should test all if-statement paths including `callable`. If we use `callable', the test should pass.

Reply:
Ok that aligns with what I was thinking. If you go to the docstring here: 
https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.binned_statistic_dd.html
You can see it says:
`‘count’ : compute the count of points within each bin. This is identical to an unweighted histogram. values array is not referenced.`
I'm not sure if we have a test on that here, if not can we add one? Presumably the way to do this would be to add something that would normally throw errors (e.g. contains Nan and/or inf but does not throw error in this case b/c is not referenced). Of course if you've (or anyone else reading) has got a better idea, please let me know. If it makes more sense for the test I'm proposing to be in a separate class that's ok too.